common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -475,4 +482,12 @@ protected AccountRecord getCachedAccountRecord(
         return targetAccount;
     }
 
+    protected boolean isMsaAccount(final MicrosoftTokenResponse microsoftTokenResponse){
+        final String tenantId = SchemaUtil.getTenantId(
```

Comment:
If we change the method to return `null` we should flip this check out so `MEGA_TENANT`.equals(tid)`